client/utils/calculatePosition.js

```diff
@@ -1,23 +1,30 @@
-export default function ({ newPosition, items, isFirstChild, insert = false }) {
+import insertActions from './insertActions';
+
+const { ADD_AFTER, ADD_BEFORE } = insertActions;
+
+export default function ({ newPosition, items, isFirstChild, action = '' }) {
```

Comment:
`action` should not be defaulted to a value since this function can be used to calculate item position when reordering, therefor we should not throw error if not provided.